src/main/java/com/cascadebot/cascadebot/commands/core/InviteCommand.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2018 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package com.cascadebot.cascadebot.commands.core;
+
+import com.cascadebot.cascadebot.commandmeta.CommandContext;
+import com.cascadebot.cascadebot.commandmeta.CommandType;
+import com.cascadebot.cascadebot.commandmeta.ICommand;
+import net.dv8tion.jda.core.entities.Member;
+
+public class InviteCommand implements ICommand {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        context.reply("Invite Link Here!");
```

Comment:
We need this to actually return the invite url ;) 

Add 
```java

    public static String getInvite() {
        return String.format("https://discordapp.com/oauth2/authorize?client_id=%s&scope=bot&permissions=%s",
                CascadeBot.INS.getSelfUser().getId(), Permission.ALL_GUILD_PERMISSIONS);
    }
```
this to CascadeBot and use in the method